dev-debian/Dockerfile

```diff
@@ -23,7 +23,8 @@ ARG SIA_DIR="/sia"
 ARG SIA_DATA_DIR="/sia-data"
 ARG SIAD_DATA_DIR="/sia-data"
 
-RUN apt-get update && apt-get install -y mime-support logrotate
+RUN apt-get update && apt-get install -y mime-support logrotate cron && \
```

Comment:
is `cron` not available? I was able to run it manually, it might be just not running but it should be available

Reply:
1. Depends on the image. I ran into the case where it wasn't installed. It doesn't hurt to make sure.
2. Once a day is mostly OK but with some log spam we might get burnt.